test/unit/active_job_iteration_test.rb

```diff
@@ -351,7 +351,7 @@ def test_active_record_job
       assert cursor
 
       assert_equal 0, ActiveRecordIterationJob.on_complete_called
-      assert_equal 2, ActiveRecordIterationJob.on_shutdown_called
+      assert_equal 4, ActiveRecordIterationJob.on_shutdown_called
```

Comment:
Any idea why this changed from 2 to 4?

Reply:
Good question. Let me check. because based on the code it shouldn't have changed the value